integration_tests/mockApis/curiousApi.ts

```diff
@@ -4,14 +4,13 @@ import { learnerEmployabilitySkills } from '../../server/data/localMockData/lear
 import { learnerEducation } from '../../server/data/localMockData/learnerEducation'
 import { LearnerProfiles } from '../../server/data/localMockData/learnerProfiles'
 import { LearnerLatestAssessmentsMock } from '../../server/data/localMockData/learnerLatestAssessmentsMock'
-import { LearnerGoalsMock, LearnerGoalsMockB } from '../../server/data/localMockData/learnerGoalsMock'
+import aValidLearnerGoals from '../../server/data/localMockData/learnerGoalsMock'
```

Comment:
`LearnerGoalsMock` and `LearnerGoalsMockB` have been refactored into `aValidLerarnerGoals` function that takes an options object to control how it is constructed (see below)